README.md

```diff
@@ -35,6 +35,7 @@ The following is a list of the force fields being considered here:
 |MDTRAJ               | 1.7.2     |
 |ParmEd               | 2.6.1     |
 |OpenForceField       | 0.1.6     |
+|Schrodinger          | 2017-2    |
```

Comment:
@ehrmanj  
If we're going to change the README we should update it to accurately reflect what we are doing. We have no actually used Schrodinger 2017-2, we were using an older version when Nam was running things and we'll switch to Desmond for OPLS2005.